MijnApp-Frontend-React/src/store/actions/user.actions.js

```diff
@@ -0,0 +1,115 @@
+import { userConstants } from "./../constants";
+import { userService } from "./../services";
+import { alertActions } from "./../actions";
+import { history } from "./../helpers";
+
+export const userActions = {
+  login,
+  logout,
+  register,
+  getAll,
+  delete: _delete,
+};
+
+function login(username, password) {
+  return (dispatch) => {
+    dispatch(request({ username }));
+
+    userService.login(username, password).then(
+      (user) => {
+        dispatch(success(user));
+        history.push("/startPage");
+      },
+      (error) => {
+        dispatch(failure(error.toString()));
+        dispatch(alertActions.error(error.toString()));
+      }
+    );
+  };
+
+  function request(user) {
```

Comment:
You can use createSlice for removing this actions and constants
https://redux-toolkit.js.org/tutorials/intermediate-tutorial#writing-the-slice-reducer